### PR TITLE
Minor Ehrlich Updates

### PR DESCRIPTION
**Pull request by danehrlich1**
1. Some changes to php.ini

- hardened some variables related to PHP sessions

2. Added httpd.conf

- can replace this with awk/sed commands later if liked 
- adds a series of  secure HTTPS headers to any web server responses
- also turns  off Server Signature, Server  Tokens, and Server ETags for further security

### CODE REVIEW

File: docker/openemr/5.0.2/Dockerfile [hunk 1]
```diff
@@ -51,6 +51,8 @@ COPY utilities/unlock_admin.php utilities/unlock_admin.sh /root/
 RUN chmod 500 run_openemr.sh autoconfig.sh /root/unlock_admin.sh \
     && chmod 000 auto_configure.php /root/unlock_admin.php
 #fix issue with apache2 dying prematurely
+RUN rm -f /etc/apache2/httpd.conf
```
Comment (jesdynf) [inline docker/openemr/5.0.2/Dockerfile]: Your COPY directive will overwrite the target -- you may remove this line.

File: docker/openemr/5.0.2/php.ini [hunk 2]
```diff
@@ -1345,14 +1345,14 @@ session.save_handler = files
 ; applications from session fixation via session adoption vulnerability. It is
 ; disabled by default for maximum compatibility, but enabling it is encouraged.
 ; https://wiki.php.net/rfc/strict_sessions
-session.use_strict_mode = 0
+session.use_strict_mode = 1
```
Comment (jesdynf) [inline docker/openemr/5.0.2/php.ini]: @bradymiller Is this permissible? Has OpenEMR been properly tested for compatibility with this flag?

Comment (bradymiller) [reply docker/openemr/5.0.2/php.ini]: hi @jesdynf , I need to do some research on this. -brady

**Comment by bradymiller** [comment]
Agree that we need to see the http.conf changes from the stock. to do this @danehrlich1  , do the following. Copy over the stock http.conf and then make that a commit. And then copy back over your modified http.conf and then make another commit. Then we will be able to look at your last commit and see the changes.

**Comment by danehrlich1** [comment]
Closing this. Will make some kind of a pull request for later.

**Pull request closed without merging**